catalog/controller/extension/payment/razorpay.php

```diff
@@ -103,10 +103,19 @@ public function index()
                 // Orders API with payment autocapture
                 $order_data = $this->get_order_creation_data($this->session->data['order_id']);
 
-                if(empty($this->session->data["razorpay_order_id_" . $this->session->data['order_id']]) === true)
+                if (isset($this->session->data["razorpay_order_amount"]) === false)
+                {
+                    $this->session->data["razorpay_order_amount"] = 0;
+                }
+
+                if ((isset($this->session->data["razorpay_order_id_" . $this->session->data['order_id']]) === false) or
+                    (isset($this->session->data["razorpay_order_id_" . $this->session->data['order_id']]) === true and
+                        ($this->session->data["razorpay_order_amount"] === 0 or
```

Comment:
Please align all conditional statement under each other in this if block

Reply:
@nikhilsharma1100 every condition should start with new line.

Reply:
@abdulwahidsharief , please use bracket for each condition for ex. 
`($this->session->data["razorpay_order_amount"] === 0)`

Reply:
@ramth05 done